webpack.config.js

```diff
@@ -112,7 +112,7 @@ module.exports = (env, argv) => {
         plugins: [
             new WebpackDashDynamicImport(),
             new webpack.SourceMapDevToolPlugin({
-                filename: '[name].js.map',
+                filename: '[file].map',
```

Comment:
caused the entrypoint sourcemap to be `main.js.map` instead of `dash_core_components.min.js.map`